src/pages/events/configure-commerce.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Install and configure Adobe I/O Events
+description: Learn how to install the Commerce modules needed for Adobe I/O Events and configure both Commerce and your existing Adobe App Builder project.
+---
+
+# Install and configure Adobe I/O Events
+
+After you've created an [App Builder project](./project-setup.md), you must install the Commerce modules that enable integrations with Adobe I/O Events.
+
+## Install Adobe I/O modules on Commerce
+
+Make the following modifications to your `composer.json` file:
+
+*  Add a `repositories` section beneath the `config` section. If the `repositories` section already exists, add the lines to the bottom of the section.
+
+   ```json
+   "repositories": {
+    "module-adobe-io-events": {
```

Comment:
Please add additional spaces before the repositories name, because `"repositories"` and `"module-adobe-io-events"` are almost on the same level
```json
"repositories": {
  "module-adobe-io-events": {
    "type": "git",
    "url": "git@github.com:magento/module-adobe-io-events.git"
  },
  "event-plugin-generator": {
    "type": "git",
    "url": "git@github.com:magento/event-plugin-generator.git"
  },
  "module-commerce-events-client": {
    "type": "git",
    "url": "git@github.com:magento/module-commerce-events-client.git"
  }
},
```